src/Raven.Server/Documents/Handlers/Admin/AdminIndexHandler.cs

```diff
@@ -81,7 +81,7 @@ private async Task PutInternal(bool validatedAsAdmin)
                     
                     if (LoggingSource.AuditLog.IsInfoEnabled)
                     {
-                        LogAuditFor(Database.Name, $"Index {indexDefinition.Name} PUT with definition: {indexToAdd}");
+                        LogAuditFor(Database.Name, "PUT", $"Index {indexDefinition.Name} with definition: {indexToAdd}");
```

Comment:
For analyzer we user '' and here we don't

Reply:
done